documentation/docs/install-pmm/install-pmm-server/baremetal/docker/restore_container.md

```diff
@@ -1,49 +1,87 @@
-# Restore container
+# Restore PMM Server container from backup
+
+You can restore PMM Server either from a standard PMM backup or from a PMM v2 backup volume that was created during migration to PMM v3. 
+
 
 !!! caution alert alert-warning "Important"
-    You must have a [backup](backup_container.md) to restore from.
+    You must have a [backup](backup_container.md) or a [PMM2 backup volume](../../../../pmm-upgrade/migrating_from_pmm_2.md#step-2-migrate-pmm-2-server-to-pmm-3) to restore from.
```

Comment:
I think it should be "Manual backup" vs "Automated backup" or "Backup to local storage" vs "Backup to another docker volume"